index.html

```diff
@@ -0,0 +1,167 @@
+
+            </table>
+
+
+
+
+          </div>
+          <div role="tabpanel" class="tab-pane" id="galery">grafika</div>
+          <div role="tabpanel" class="tab-pane" id="map">mapa</div>
+        </div>
+
+      </div>
+      <div class="col-md-4 ">
+        <canvas id="myChart"></canvas>
+        <canvas id="categoryCHart">
+        </canvas>
+      </div>
+    </div>
+
+
+
+  </div>
+
+  <div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title ratesModalTitle" id="myModalLabel">Modal title</h4>
+        </div>
+        <div class="modal-body ratesModalBody">
+          ...
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-default" data-dismiss="modal">Zamknij</button>
+          <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addRate">Dodaj Ocene</button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+
+  <div class="modal fade" id="addRate" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title" id="myModalLabel">Dodaj ocene</h4>
+        </div>
+        <div class="modal-body">
+          <form id="addRateForm">
+            <div class="form-group">
+              <label for="user">Użytwkonik</label>
+              <input type="text" class="form-control" id="user" name="user" placeholder="Nazwa Uzytkowniak">
+              <br>
+              <label for="email">Adres email</label>
+              <input type="email" class="form-control" id="email" name="email" placeholder="email">
+              <br>
+              <label for "date">Data</label>
+              <input type="text" class="form-control" id="datepicker">
+              <br>
+              <label for="rate">Ocena</label>
+              <input type="text" class="form-control" id="rate" name="rate" placeholder="ocena">
+              <br>
+              <label for="contentRate">Tresc</label>
+              <input type="text" class="form-control" id="contentRate" name="contentRate" placeholder="tresc">
+            </div>
+
+          </form>
+        </div>
+        <div class="modal-footer">
+          <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          <button type="button" class="btn btn-primary">Wyślij</button>
+        </div>
+      </div>
+    </div>
+  </div>
+
+
+  <footer/>
+</body>
+
+</html>
```

Comment:
na końcu plików *zawsze* robimy nową linię (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)
najlepiej ustawić sobie w edytorze, żeby sam ją dodawał na zapisie pliku